app/Console/Commands/LdapSync.php

```diff
@@ -390,6 +391,12 @@ public function handle()
                         $user->location_id = $location->id;
                     }
                 }
+                //updates assets location based on user's location
+                $assets = Asset::where('assigned_to', '=', $user->id)->get();
+                    foreach($assets as $asset){
```

Comment:
Have we tested this with very large data sets? Foreaching through large sets can timeout. We may want to consider a mass update or eloquent directly.

Reply:
I replaced it with an eloquent update:
` Asset::where('assigned_to', '=', $user->id)->update(['location_id' => $user->location_id]);`